src/mgr/ActivePyModules.cc

```diff
@@ -21,6 +21,8 @@
 
 #include "osd/OSDMap.h"
 #include "mon/MonMap.h"
+#include "osd/OSDMap.cc"
```

Comment:
why do we need to include `osd/OSDMap.cc` here?

Reply:
because we need to quote the method defined in class OSDUtilizationFormatDumper and the class was directly defined in OSDMap.cc.

Reply:
> why do we need to include `osd/OSDMap.cc` here?

because we need to quote the method defined in class OSDUtilizationFormatDumper and the class was directly defined in OSDMap.cc.